src/indicators/design-guidelines.md

```diff
@@ -0,0 +1,22 @@
+The following is a brief set of design guidelines for indicators:
+
+## Calculators
+
+ - calculators should operate on an input array of data, transforming it into a new array containing the indicator output values.
+ - calculators should expose a `value` accessor property (or a number of accessors if required), which is used to extract values from the source array. For calculators that only depend on a single value for their input this should be the identity function.
+ - calculators should not have to handle undefined values in the input array of data.
+ - the length of the output array should be the same as the input array.
+ - if a calculator creates 'undefined' values, the value used should be configurable.
```

Comment:
If _undefined_ is referring to the JavaScript type, is it worth surrounding it with backticks, to show it refers to code?